modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -134,6 +134,20 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 // CCFR here.
 
+=== Heading formatting would be partially applied to the content within the `summary` element when the caret was positioned between words.
+// #TINY-10312
+
+With the release of {productname} 6.8, applying heading formatting to the content within the `<summary>` element was introduced. However, an issue was identified: 
+
+- if the caret was positioned between words, and;
+- before or after a **`+&nbsp;+`** space character.
```

Comment:
This should not be two list items here. It is just one condition, namely, the caret to be positioned between the words.
When I described the conditions for issue to reproduce
> ... if the caret was positioned between words (before or after a space character) 

I meant the cases  `Hello| word` or `Hello |world`. So the caret being between the words can be at the end of first word or at the beginning of second word.

Reply:
make some improvements. feel free to resolve this conversation if you are happy with the new changes.